Druid/DRBytecodeCompilerCompiler.class.st

```diff
@@ -2,7 +2,8 @@ Class {
 	#name : #DRBytecodeCompilerCompiler,
 	#superclass : #DRAbstractCompilerCompiler,
 	#instVars : [
-		'primitiveCompilerName'
+		'primitiveCompilerName',
+		'failedBytecodes'
```

Comment:
The compiler is not designed to be reused... So it could only have one failed bytecode, not many. If we try to reuse it, it will potentially generate problems.